package.json

```diff
@@ -24,8 +24,7 @@
     "webpack": "^1.12.9"
   },
   "scripts": {
-    "update:nodejs": "sudo rm -rf ~/.nvm - curl -sL 'https://deb.nodesource.com/setup_${NODE_RELEASE}' | sudo -E bash && sudo apt-get install -y nodejs",
```

Comment:
are you sure you can drop this?

Reply:
i found a way to install a specific node_js version next to php.